lib/init.js

```diff
@@ -38,7 +38,7 @@ function _init (opts, tracker, cb) {
       return cbPassedIn ? cb(err) : setImmediate(() => tracker.emit('error', err))
     }
 
-    setImmediate(() => tracker.emit('done', result))
```

Comment:
`setImmediate` is not needed here because unlike `error` case, we'll reach at this point only after enough delay so that the consumer would get time to bind the events.

and the for removing is so `done` fires instantly and progress tracker gets a chance to clear the cli progress/spinner **before** anyone writes anything in the output.